config/config_types.go

```diff
@@ -78,6 +78,7 @@ var (
 // Exchanges
 type Config struct {
 	Name              string                  `json:"name"`
+	DataDir           string                  `json:"datadir,omitempty"`
```

Comment:
DataDir -> DataDirectory; 
`json:"datadir -> `json:"dataDirectory`
I think it might be better to remove omitempty, just so it's more obvious for people to set. What do you think?

Reply:
Also, config_example.json needs to be updated with this field as well.

Reply:
1. Renames, done.
2. remove omitempty, done.
3. For config_example.json, Not sure what to put there, the default (in user home) would be used if not set. If I set a relative directory, then it depends on the working directory of the process. Another (more intuitive?) option would be to change behaviour to resolve it to a relative directory of the config file?